content/articles/adding-domain.markdown

```diff
@@ -7,18 +7,18 @@ categories:
 
 # Adding a Domain
 
-When adding a domain to your account, you can choose to register, transfer or just use our DNS service if the domain is already registered with another registrar.
+When adding a domain to your account, you can choose to register, transfer, or just use our DNS service if the domain is already registered with another registrar.
 
 <div class="section-steps" markdown="1">
 ##### To add a domain
 
-1.  Log into DNSimple with your user credentials.
-1.  If you have more than one account, select the relevant one.
-1.  On the header, click the <label>Domains</label> tab
+1.  Log in to DNSimple with your user credentials.
+1.  Select the relevant account.
```

Comment:
A customer with a single account (which is the most likely case) will not have to select an account, thus if they try to actually follow the steps with the change here, they will stop at this point because there is no way to select an account.